packages/@react-stately/toast/src/useToastState.ts

```diff
@@ -134,19 +134,15 @@ export class ToastQueue<T> {
       }
     }
 
-    if (toast.priority) {
-      this.queue.splice(low, 0, toast);
-    } else {
-      this.queue.unshift(toast);
-    }
+    this.queue.splice(low, 0, toast);
```

Comment:
Is this a breaking change or was `this.queue.unshift(toast);` a bug we hadn't found?

Reply:
This reverts the logic to what is currently released

Reply:
Right, I just remembered that and was coming back to delete it, but you responded.